.storybook/preview.tsx

```diff
@@ -115,14 +120,18 @@ const withThemeSwitcher: Decorator = (
         return (
             <RenderStateRoot>
                 <ThemeSwitcherContext.Provider value={theme}>
-                    <Story />
+                    <div ref={containerRef}>
+                        <Story />
+                    </div>
                 </ThemeSwitcherContext.Provider>
             </RenderStateRoot>
         );
     }
     return (
         <ThemeSwitcherContext.Provider value={theme}>
-            <Story />
+            <div ref={containerRef}>
```

Comment:
Idea: What do you think about making a different decorator for the announcer so that it is separated from the theme switcher? (not blocking)

Reply:
Good call. I pushed a change with a separate decorator!